packages/jest-types/src/Transform.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {Script} from 'vm';
+import {Path, ProjectConfig} from './Config';
+
+export type TransformedSource = {
+  code: string;
+  map?:  // copied from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/363cdf403a74e0372e87bbcd15eb1668f4c5230b/types/babel__core/index.d.ts#L371-L379
```

Comment:
Is it an option to depend on `@types/babel__core` and make this
```
map?: BabelFileResult['map'] | string | null;
```

Or do we want to keep `@jest/types` free of any deps at all

Reply:
Good call! Can actually get it from `source-map`, which is nicer: https://github.com/mozilla/source-map/blob/ac518d2f21818146f3310557bd51c13d8cff2ba8/source-map.d.ts#L1-L12